src/internal/group_api/mod.rs

```diff
@@ -172,6 +177,10 @@ impl GroupGetResult {
     pub fn member_list(&self) -> Option<&Vec<UserId>> {
         self.member_list.as_ref()
     }
+    /// None if the calling user is not a group admin
```

Comment:
Should explain the other two states: Some(true), Some(false)

Reply:
I was/am having difficulty wording it succinctly. I made an attempt at a comment, but I am open to suggested changes.